core/domain/feedback_services.py

```diff
@@ -75,7 +97,22 @@ def create_message(
     """Creates a new message for the thread and subscribes the author to the
     thread.
 
-    Returns False if the message with the ID already exists.
+    Args:
+        exploration_id: str. The exploration id the thread belongs to.
+        author_id: str. The author id who creates this message.
+        updated_status: str, one of STATUS_CHOICES. New thread status.
+            Must exist in the first message of a thread. For the
+            rest of the thread, should exist only when the status changes.
+        updated_subject: str. New thread subject. Must exist in the first
+            message of a thread. For the rest of the thread, should exist
```

Comment:
Must be supplied if this is the first message of a thread, or if the current message includes a change in the thread subject.